openshift/templates/nodejs.json

```diff
@@ -189,6 +189,26 @@
       "description": "A secret string used to configure the GitHub webhook",
       "generate": "expression",
       "from": "[a-zA-Z0-9]{40}"
+    },
+    {
+      "name": "DATABASE_SERVICE_NAME",
+      "description": "Database service name"
+    },
+    {
+      "name": "MONGODB_USER",
+      "description": "Username for MongoDB user that will be used for accessing the database"
+    },
+    {
+      "name": "MONGODB_PASSWORD",
+      "description": "Password for the MongoDB user"
+    },
+    {
+      "name": "MONGODB_DATABASE",
+      "description": "Database name"
+    },
+    {
+      "name": "MONGODB_ADMIN_PASSWORD",
+      "description": "Password for the database admin user"
```

Comment:
ok.  As long as the actual application behavior use/not use the DB is based on whether _SERVICE_HOST is defined, rather than whether DATABASE_SERVICE_NAME is defined.